packages/printable-itinerary/src/access-leg.js

```diff
@@ -38,7 +47,16 @@ export default function AccessLeg({ config, leg, LegIcon }) {
 }
 
 AccessLeg.propTypes = {
+  accessibilityScoreGradationMap: PropTypes.shape({
+    color: PropTypes.string,
+    text: PropTypes.string,
+    icon: PropTypes.element
+  }),
```

Comment:
Extract this shape into a var, export (out of `index.js` I presume), and reuse in this package. In light of TypeScript-ing this component in some near future, I would skip adding the var to `core-utils/types`.

Reply:
This is definitely cleaner, but for some reason the PropType export caused a circular dependency which both failed linting and caused the story shots to break.